dashboard/src/components/entitypage/elements/PrometheusGraph.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useEffect } from "react";
+import { Chart } from "chart.js";
+import { connect } from "react-redux";
+
+const PrometheusGraph = ({ query, time, timeRange }) => {
+  useEffect(() => {
+    var myChart = new Chart(chartRef.current, {
+      type: "line",
+
+      plugins: [require("chartjs-plugin-datasource-prometheus")],
+      options: {
+        maintainAspectRatio: false,
+        responsive: true,
+        plugins: {
+          "datasource-prometheus": {
+            prometheus: {
+              endpoint: "http://localhost:9090",
+            },
+            query: `${query}`,
+            timeRange: {
+              type: "relative",
+
+              start: -parseInt(timeRange.timeRange[0]) * 60 * 1000,
```

Comment:
Can you make these numbers, named constant, also why is this negative?

Reply:
type "relative" means relative to time.now, in the past or the future, so historical data would be negative values.